### DISCUSSION TITLE
Cannot read property '_nextBubbleNoFallback' of undefined

### DISCUSSION THREAD

**Discussion by chngjunjie** [vercel/next.js#35099] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

Macos , 
@hapi/hapi": "^20.1.0",
next": "^10.0.9",

### What browser are you using? (if relevant)

Chrome

### How are you deploying your application? (if relevant)

#!/bin/bash

trap "exit" INT

set -ev
# ls ./node_modules/.bin
npm run git:hash
npm run build:cssmodule
npm run build:next
npm run build:server

ENV=local nodemon

### Describe the Bug

error - TypeError: Cannot read property '_nextBubbleNoFallback' of undefined
    at Object.fn (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/next/server/next-server.ts:997:39)
    at Router.execute (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/next/server/router.ts:346:40)
    at async DevServer.run (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/next/server/next-server.ts:1249:6)
    at async DevServer.run (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/next/dist/server/dev/next-dev-server.js:355:20)
    at async DevServer.handleRequest (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/next/dist/server/next-server.js:297:20)
    at async /Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/src/main/server/nextjs/next-wrapper.ts:7:5
    at async exports.Manager.execute (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/@hapi/hapi/lib/toolkit.js:60:28)
    at async Object.internals.handler (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at async exports.execute (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at async Request._lifecycle (/Users/Allen_Chng/Desktop/xb_project/sm-exbooking/vanellope/node_modules/@hapi/hapi/lib/request.js:371:32)
    
So basically my application is running Next with hapi. After build application with next and run, it going to localhost and show me this error. 

There's few quick i research from google. I found out the similar case, but din explain how it cause . [github discussion]( https://github.com/vercel/next.js/discussions/25436) . limited resource from google.

### Expected Behavior

Run without this error

### To Reproduce

I not sure how it cause the error, at the first time i run without error , but it happen somehow after build few time.
